chain-impl-mockchain/src/date.rs

```diff
@@ -121,7 +121,8 @@ impl str::FromStr for BlockDate {
 mod tests {
```

Comment:
Same here, this probably does not need the `cfg(feature = ...)` alternative at all. And everywhere else where the mod is private.